rpcd/playbooks/ansible.cfg

```diff
@@ -12,19 +12,19 @@ timeout = 120
 
 # Set the path to the folder in os-ansible-deployment which holds the dynamic
 # inventory script - new config setting for ansible v1.9 and above
-inventory = /opt/os-ansible-deployment/playbooks/inventory/
+inventory = ../../os-ansible-deployment/playbooks/inventory/
```

Comment:
maybe discover the absolute path of the basedir and have everything relative to that. Then there is no hard set requirement to clone it to any specific place, then you can pushd and popd around as you feel like. eg:

```
DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd ../ && pwd )
OSAD_DIR="${DIR}/os-ansible-deployment'"
RPCD_DIR="${DIR}/rpcd"
```

etc